src/styles/card.css

```diff
@@ -0,0 +1,96 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --primary-color: #00acdc;
+  --main-text-color: #060b35;
+  --text-secondary-color: #616070;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-secondary-color);
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  height: 134px;
```

Comment:
do not set height eplicitly. what if you have more lines of text? it will not fit and the container won't stretch

Reply:
Okay understood. Thank you.

Reply:
Hi @ihor-jpeg Isn't dimensions of the image fixed. I had to set the width and height of the image as it doesn't contain text not for the card itself.